model-optimizer/extensions/back/ReverseInputChannels.py

```diff
@@ -94,8 +94,36 @@ class ReverseChannelsPropagationDown(BackReplacementPattern):
 
         'Shape': lambda node, rc: ReverseChannelsPropagationDown.pass_rc_through_shape(node, rc),
         'ShapeOf': lambda node, rc: ReverseChannelsPropagationDown.pass_rc_through_shape(node, rc),
+
+        'Pad': lambda node, rc: ReverseChannelsPropagationDown.pass_rc_through(node, rc),
     }
 
+    @staticmethod
+    def pass_rc_through(node: Node, reverse_channels: Node):
+        r"""
+        BEFORE                          AFTER
+
+          previous_op
+              |
+        ReverseChannels begin end       previous_op begin end
+                     \    /   /                   \   /  /
+                        Pad                         Pad
+                                                     |
+                                              ReverseChannels
```

Comment:
It is not necessarily for `Pad` operation. Please draw a graph for arbitrary node